docs/podman-search.1.md

```diff
@@ -0,0 +1,107 @@
+% podman(1) podman-search - Tool to search registries for an image
+% Urvashi Mohnani
+# podman-search "1" "January 2018" "podman"
+
+## NAME
+podman search - Search a registry for an image
+
+## SYNOPSIS
+**podman search**
+**TERM**
+[**--filter**|**-f**]
+[**--format**]
+[**--limit**]
+[**--no-trunc**]
+[**--registry**]
+[**--help**|**-h**]
+
+## DESCRIPTION
+**podman search** searches a registry or a list of registries for a matching image.
+The user can specify which registry to search by setting the **--registry** flag, default
+is the default registries set in the config file.
+The number of results can be limited using the **--limit** flag. If more than one registry
+is being searched, the limit will be applied to each registry. The output can be filtered
+using the **--filter** flag.
+
+**podman [GLOBAL OPTIONS]**
+
+**podman search [GLOBAL OPTIONS]**
+
+**podman search [OPTIONS] TERM**
+
+## OPTIONS
+
+**--filter, -f**
+Filter output based on conditions provided (default [])
+
+Supported filters are:
+- stars (int - number of stars the image has)
+- is-automatec (boolean - true | false) - is the image automated or not
+- is-official (boolean - true | false) - is the image official or not
+
+**--format**
+Change the putput format to a Go template
+
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder** | **Description**              |
+| --------------- | ---------------------------- |
+| .Index          | Registry                     |
+| .Name           | Image name                   |
+| .Descriptions   | Image description            |
+| .Stars          | Star count of image          |
+| .Official       | "[OK]" if image is official  |
+| .Automated      | "[OK]" if image is automated |
+
+**--limit**
+Limit the number of results
+Note: The results of each registry will be limited to this value.
```

Comment:
suggests "results of" to "results from".  If you have 10 results in a registry and limit is set to 3, which 3 do you get?  Oldest, newest, alphabetical, other?  It would probably be nice to talk about ordering here.  Might also be nice to have a way to sort the output.

Reply:
There is no particular order, just the order in which the api returns the results. docker search sorts it based on the star count, highest to lowest. I didn't bother to do that as that would make search slower, and it is already a bit slow. I don't think podman should care that much about the star count, looks to be mainly a docker hub thing. @rhatdan what are your thoughts? I can definitely clarify that the output is ordered based on the order of the results from the api.

Reply:
Lets not sort for now, until someone complains.